Symfony/CS/Fixer/PSR2/MethodArgumentSpaceFixer.php

```diff
@@ -53,12 +53,20 @@ public function fix(\SplFileInfo $file, $content)
     }
 
     /**
+     * {@inheritdoc}
+     */
+    public function getDescription()
+    {
+        return 'In method arguments and method call, there MUST NOT be a space before each comma and there MUST be one space after each comma.';
+    }
+
+    /**
      * Method to insert space after comma and remove space before comma.
      *
-     * @param Tokens $tokens tokens to handle
-     * @param int    $index  index of token
+     * @param Tokens $tokens
+     * @param int    $index
      */
-    public function fixSpace(Tokens $tokens, $index)
+    private function fixSpace(Tokens $tokens, $index)
```

Comment:
Changing the visibility is a BC break. So there are 2 questions to ask here:

- does this method already exist in the stable release (1.3) ? If yes, this change must be delayed until 2.0 because it breaks BC; end of the checklist.
- does this method exist in the (not yet tagged) 1.4 branch ? If yes, it should be fixed there (before the 1.4 branch becomes stable, making it become the first checklist item)